sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -543,13 +543,15 @@ export class ApplicationStudentsController extends BaseController {
       a.disbursementDate < b.disbursementDate ? -1 : 1,
     );
     const [firstDisbursement, secondDisbursement] = disbursements;
+    const [scholasticStandingChange] = application.studentScholasticStandings;
     return {
       applicationStatusUpdatedOn: application.applicationStatusUpdatedOn,
       pirStatus: application.pirStatus,
       firstCOEStatus: firstDisbursement?.coeStatus,
       secondCOEStatus: secondDisbursement?.coeStatus,
       exceptionStatus: application.applicationException?.exceptionStatus,
       appealStatus,
+      scholasticStandingChangeType: scholasticStandingChange?.changeType,
```

Comment:
are we returning only the first scholasticStanding from the list?

Reply:
Yes, in the same way as `pirStatus`, `appealStatus`, `exceptionStatus`.